frontend/smart-kickers-game/src/App.js

```diff
@@ -1,5 +1,6 @@
 import { useEffect, useState } from 'react';
 import './App.css';
+import * as GameAPI from './apis/Game';
```

Comment:
In my opinion, if you import only one thing and use it once, it's easier to use a `named import`.